assets/training/finetune_acft_hf_nlp/src/utils/finetune/optimization_utils.py

```diff
@@ -0,0 +1,8 @@
+from azureml.acft.contrib.hf.nlp.constants.constants import Tasks
```

Comment:
Unused code as we do not copy the entire src folder in finetune component. Will fix this in coming PRs